### ISSUE TITLE
Invalid string length error if used with babel-plugin-react-intl-auto

### ISSUE THREAD

**Issue by matej-prokop** [akameco/extract-react-intl-messages#14]
Hi! I am not able to make `extract-react-intl-messages` working with `babel-plugin-react-intl-auto`.

My `.babelrc` has following content:
```
{
  "presets": [
    [ "env", { "modules": false, "loose": true } ],
    "react"
  ],
  "plugins": [
    "transform-object-rest-spread",
    [ "transform-class-properties", { "loose": true } ],
    [ "react-intl-auto", { "filebase": true } ]
  ]
}
```

I execute messages extraction via `node ./node_modules/extract-react-intl-messages/cli.js -l=en -o translations 'src/**/*.js'` 

If any of my .js files has message without `id` example:  `<FormattedMessage defaultMessage="Try again" />` then extraction ends up with:
```
RangeError: Invalid string length
    at JSON.stringify (<anonymous>)
    at then.str (/Users/matejprokop/dev/react-components/node_modules/write-json-file/index.js:40:22)
    at <anonymous>
```

If I add missing id to the message definition. Then extraction script works as expected and produce `translations/en.json`

- `node` version: v8.9.2
- `npm` (or `yarn`) version: 5.5.1

Probably, I overlooked some important detail. Let me know if you need more info. Thank you!

**Comment by akameco** [comment]
@matej-prokop Thank you for your reporting :)
Hmm..., Could you show me reproduction repository?

**Comment by matej-prokop** [comment]
Of course :)

Here is repo https://github.com/matej-prokop/extract-react-intl-messages-14-repro. Please follow steps to reproduce in README.md

**Comment by matej-prokop** [comment]
@akameco In a meantime, I found out that if you replace in `.babelrc`

```
    [
      "react-intl-auto",
      {
        "filebase": false
      }
    ]
```

by

```
    [
      "react-intl-auto",
      {
        "filebase": false,
        "removePrefix" : true
      }
    ]
```

then it works without any errors.

**Comment by capaj** [comment]
I've hit this as well today on node 9. Even after changing the `"removePrefix" : true` I still get that crash. It is quite possible I have something badly configured because this is first time I am trying to set up intl with react-intl-auto.

**Comment by solomon23** [comment]
My pull https://github.com/akameco/extract-react-intl-messages/pull/29 fixed this error for me

**Comment by akameco** [comment]
Thank you @solomon23 :)
Released v0.11.2. Please try updating.
